src/chainstate/coordinator/tests.rs

```diff
@@ -3347,9 +3348,17 @@ fn test_sbtc_peg_in_btc_op() {
         let expected_winner = good_op.txid();
         let mut ops = vec![good_op];
         let peg_wallet_address = PoxAddress::Addr32(false, PoxAddressType32::P2TR, [0; 32]);
+        let recipient_btc_address = PoxAddress::Standard(stacker.into(), None);
+        let canonical_chain_tip_snapshot =
+            SortitionDB::get_canonical_burn_chain_tip(sort_db.conn()).unwrap();
+
+        let chain_tip = StacksBlockId::new(
+            &canonical_chain_tip_snapshot.consensus_hash,
+            &canonical_chain_tip_snapshot.winning_stacks_block_hash,
+        );
 
         if ix == 0 {
```

Comment:
nit: `match ix {`

Reply:
Of course, thanks for spotting! That's what I get for cargo-cult-copying the structure of similar test cases in this file :sweat_smile:

Reply:
Updated :heavy_check_mark: